.gitignore

```diff
@@ -6,3 +6,6 @@ build/
 /controllers/teleport_controller/teleport_controller
 /nugus_controller_test*log.txt
 *.log
+.vs/slnx.sqlite
+.vs/VSWorkspaceState.json
+.vs/Webots/v16/.suo
```

Comment:
I'm not sure if we really want to add this in - I don't think it's practical to add in files generated from IDEs unless it's popular, but Visual Studio not part of the normal NUbots workflow.

Reply:
I don't think it's too bad. But I'd add only a single line, to ignore the entire .vs folder:

```
.vs
```